app/admin/products.rb

```diff
@@ -0,0 +1,111 @@
+include UserUtils
+
+ActiveAdmin.register Product do
+  config.create_another = true
+  actions :all, except: [:destroy]
+  permit_params :id, :user_email, :create_another
+
+  # To do: set up index file export
+  index download_links: false do
+    id_column
+    column 'Product Name', :name
```

Comment:
```suggestion
    column 'Owner email', :user_email
    column 'Created at', :created_at
    column 'Updated at', :updated_at
    actions
```
This is a little too minimal right now. Let's add more information to the index and the basic set of actions.